From 25b2c21b8ff02f8e36ac6f333be4df5206f0b975 Mon Sep 17 00:00:00 2001
From: Xinyu Chen <xinyu.chen@freescale.com>
Date: Wed, 29 Aug 2012 11:03:05 +0800
Subject: [PATCH 215/280] ENGR00221837-2 ipu lib: only build for i.MX5
 platform

We do not use IPU user space lib anymore for i.MX6

Signed-off-by: Xinyu Chen <xinyu.chen@freescale.com>
---
 ipu/Android.mk | 5 ++++-
 1 file changed, 4 insertions(+), 1 deletion(-)

diff --git a/ipu/Android.mk b/ipu/Android.mk
index 0ca1fae..2e33736 100644
--- a/ipu/Android.mk
+++ b/ipu/Android.mk
@@ -1,3 +1,5 @@
+ifeq ($(BOARD_SOC_CLASS),IMX5X)
+ifeq ($(HAVE_FSL_IMX_IPU),true)
 LOCAL_PATH := $(call my-dir)
 include $(CLEAR_VARS)
 
@@ -16,4 +18,5 @@ LOCAL_LD_FLAGS += -nostartfiles
 LOCAL_PRELINK_MODULE := false
 LOCAL_MODULE_TAGS := eng
 include $(BUILD_SHARED_LIBRARY)
-
+endif
+endif
-- 
1.8.0

